seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/sql/SQLTransform.java

```diff
@@ -70,22 +69,6 @@ public SQLTransform(@NonNull ReadonlyConfig config, @NonNull CatalogTable catalo
         } else {
             this.engineType = ZETA;
         }
-
-        List<String> sourceTableNames = config.get(CommonOptions.SOURCE_TABLE_NAME);
-        if (sourceTableNames != null && !sourceTableNames.isEmpty()) {
-            this.inputTableName = sourceTableNames.get(0);
-        } else {
-            this.inputTableName = catalogTable.getTableId().getTableName();
-        }
```

Comment:
we should keep this at now. The logs displayed by the two different names will be slightly different.

Reply:
done